pkg/provider/provider.go

```diff
@@ -86,13 +78,122 @@ func configure(version string, p *schema.Provider) func(context.Context, *schema
 		ba_bearer_token := schema.Get("ba_bearer_token").(string)
 		ba_api_uri := schema.Get("ba_api_uri").(string)
 		// set our meta to be a new api.API
-		// this can be turned into concrete clients
+		// this can be turned into concrete cba_api_uri := schema.Get("ba_api_uri").(string)lients
```

Comment:
This line is probably a copy paste error, right?

Reply:
sure, fixed